src/main/java/org/fundacionjala/coding/nestor/Customer.java

```diff
@@ -0,0 +1,77 @@
+package org.fundacionjala.coding.nestor;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+public class Customer {
+    private String _name;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_name" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)